Makefile

```diff
@@ -101,6 +109,15 @@ test/e2e/workflow:
 		--kubeconfig ./cluster/.kubeconfig \
 		--go-test-flags "$(E2E_TEST_ARGS)"
 
+test/e2e/lifecycle:
+	$(OPERATOR_SDK) test \
+		local \
+		./test/e2e/lifecycle \
```

Comment:
$@ even to same in th other, we can even merge the too into a more general test/e2e/*.

Reply:
Done